tests/phpunit/CRM/Batch/Form/EntryTest.php

```diff
@@ -97,18 +104,16 @@ public function setUp(): void {
       'name_b_a' => 'Test Employer of',
     ];
     $this->_relationshipTypeId = $this->relationshipTypeCreate($params);
-    $this->_orgContactID = $this->organizationCreate();
-    $this->_financialTypeId = 1;
-    $this->_membershipTypeName = 'Mickey Mouse Club Member';
+    $this->_orgContactId = $this->organizationCreate();
     $params = [
-      'name' => $this->_membershipTypeName,
+      'name' => self::MEMBERSHIP_TYPE_NAME,
```

Comment:
We don't actually need this constant do we - neither this or the financialType one appear to be accessed elsewhere?

Reply:
I don't feel strongly as to whether we leave as consts or not. I'll change.